packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -94,6 +94,23 @@
                     </oc-button>
                   </div>
                 </li>
+                <li
+                  v-for="(mimetype, key) in getMimeTypes()"
+                  v-if="mimetype.allow_creation === true"
+                  :key="key"
+                >
+                  <div>
+                    <oc-button
+                      appearance="raw"
+                      justify-content="left"
+                      :class="['uk-width-1-1']"
+                      @click="showCreateResourceModal(false, mimetype.ext, false, true)"
+                    >
+                      <oc-icon :name="mimetype.icon || 'file'" />
+                      <span>{{ 'New ' + mimetype.name }}</span>
```

Comment:
This will not get translated. Please do either of the following:
a) use `v-translate` and interpolate the mimetype name into the string
b) create a method for building the button label, use `this.$gettext` and `this.$gettextInterpolate` in that method with the same intention as in (a) and use that method here with the mimetype name as param.

Reply:
```html
<translate :translate-params="{ name: mimetype.name }"
  >New %{name}</translate
>
```